app/controllers/api/account.php

```diff
@@ -2293,7 +2296,11 @@ function sendSessionAlert(Locale $locale, Document $user, Document $project, Doc
 
         $subject = $locale->getText("emails.otpSession.subject");
         $preview = $locale->getText("emails.otpSession.preview");
+        $heading = $locale->getText("emails.otpSession.heading");
+
         $customTemplate = $project->getAttribute('templates', [])['email.otpSession-' . $locale->default] ?? [];
+        $smtpBaseTemplate = $project->getAttribute('smtpBaseTemplate', 'email-base');
+        $bodyTemplate = __DIR__ . '/../../config/locale/templates/' . $smtpBaseTemplate . '.tpl';
```

Comment:
I tend to agree with AI here, there is potential for abuse in future. Lets check if there is some PHP method that can help us protect it here, something that disallows changing directories, stuff like ../../../ and such. So we are certain we only got file name here